packages/eslint-plugin-template/src/utils/get-original-attribute-name.ts

```diff
@@ -0,0 +1,63 @@
+import type {
+  TmplAstBoundAttribute,
+  TmplAstBoundEvent,
+  TmplAstTextAttribute,
+} from '@angular/compiler';
+import { BindingType, ParsedEventType } from '@angular/compiler';
+
+// TODO: It seems like `TmplAstBoundAttribute.type` is incorrectly typed at https://github.com/angular/angular/blob/2d1347b2ceef8c1e564c0f7cfe3a0a740b915e80/packages/compiler/src/render3/r3_ast.ts#L64.
+type BoundAttribute = Omit<TmplAstBoundAttribute, 'type'> &
```

Comment:
`TmplAstBoundAttribute.type` is declared as `BindingType` (as you can see [here](https://github.com/angular/angular/blob/2d1347b2ceef8c1e564c0f7cfe3a0a740b915e80/packages/compiler/src/render3/r3_ast.ts#L64)), however its type should be `BoundAttribute`, no?

The same is valid for `TmplAstBoundEvent` below.

Reply:
Question for the Angular Team that one :)w

Reply:
I'll open an issue there asap... for now I've simplified this method as I noticed that we can rely on `.keySpan.name` to get the full name (except in two-way data binding cases, which are already being handled properly).

Reply:
Are you wanting this PR to be reviewed now then?

Reply:
OMG, I totally missed your message here. Sorry about that, as I changed this implementation, the AST point doesn't matter anymore... so you can revise it whenever you want :)